public/docs/ts/latest/cookbook/i18n.jade

```diff
@@ -112,48 +211,68 @@ code-example(language="sh" class="code-shell").
 code-example(language="sh" class="code-shell").
   ./node_modules/.bin/ng-xi18n
 
+.l-sub-section
+  :marked
+    Windows users may have to quote the command: like this `"./node_modules/.bin/ng-xi18n"`
+
 :marked
   By default the tool generates a translation file named **`messages.xlf`** in the
   <a href="https://en.wikipedia.org/wiki/XLIFF" target="_blank">XML Localisation Interchange File Format (XLIFF, version 1.2)</a>.
 
-code-example(language="sh" class="code-shell").
-  ./node_modules/.bin/ng-xi18n  --i18nFormat=xmb
-
-.l-sub-section
-  :marked
-    Windows users may have to quote the command:
-  code-example(language="sh").
-    "./node_modules/.bin/ng-xi18n"
-  :marked
-    Consider adding a convenience shortcut to the `scripts` section of the `package.json`
-    to make the command easier to remember and run:
-  code-example(format='.').
-    "scripts": {
-      "i18n": "ng-xi18n", 
-      ...
-    }
-  :marked
-    Now you can enter:
-  code-example(language="sh" class="code-shell").
-    npm run i18n
-
+a#other-formats
 :marked
   ### Other translation formats
 
   You can generate a file named **`messages.xmb`** in the 
   <a href="http://cldr.unicode.org/development/development-process/design-proposals/xmb"  target="_blank">XML Message Bundle (XMB)</a> format
-  by adding the `--i18nFormat=xmb` switch.
+  by adding the `--i18nFormat=xmb` flag.
+
+code-example(language="sh" class="code-shell").
+  ./node_modules/.bin/ng-xi18n  --i18nFormat=xmb
 
+:marked
   This sample sticks with the _XLIFF_ format.
 
+a#ng-xi18n-options
+:marked
+  ### Other options
+  You may have to specify additional options. 
+  For example, if your `tsconfig.json` TypeScript configuration file is located other than in the root folder,
+  you must identify the path to it with the `-p` option:
+code-example(language="sh" class="code-shell").
+  ./node_modules/.bin/ng-xi18n -p path/to/tsconfig.json
+  ./node_modules/.bin/ng-xi18n  --i18nFormat=xmb -p path/to/tsconfig.json
+
+
+a#npm-i18n-script
+:marked
+  ### Add an _npm_ script for convenience
+  
+  Consider adding a convenience shortcut to the `scripts` section of the `package.json`
+  to make the command easier to remember and run:
+code-example(format='.' language='sh' ).
+  "scripts": {
+    "i18n": "ng-xi18n", 
+    ...
+  }
+:marked
+  Now you can issue command variations such as these:
+code-example(language="sh" class="code-shell").
+  npm run i18n
+  npm run i18n -- -p path/to/tsconfig.json
```

Comment:
I mean you never want to write `npm run i18n -- -p path/to/tsconfig.json` but `npm run i18n` and put the flag in your package.json